app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java

```diff
@@ -39,6 +40,10 @@
     @Column(name = "operation_type")
     public String operation_type;
 
+    private static Context mContext = Home.getContext();
+
+    private static PebbleSync pebbleSync = new PebbleSync(mContext);
```

Comment:
No. When I did that it created a PebbleSync every time a BgReadind was sent to the queue. The only thing we need to do at that code area is sendData() if we are to send to the pebble. 

 
John Stevens
M - +61 418526349

> On 20 May 2015, at 00:09, Stephen Black <notifications@github.com> wrote:
> 
> In app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java:
> 
> > @@ -39,6 +40,10 @@
> >      @Column(name = "operation_type")
> >      public String operation_type;
> >  
> > +    private static Context mContext = Home.getContext();
> > +
> > +    private static PebbleSync pebbleSync = new PebbleSync(mContext);
> this should be inside the function with all the rest(on line 117)
> 
> —
> Reply to this email directly or view it on GitHub.
>